### ISSUE TITLE
Regularization of Neural Networks using DropConnect

### ISSUE THREAD

**Issue by mingqook** [mingqook/Reading_Paper#48]
## Abstract (초록) 📝
We introduce DropConnect, a generalization of DropOut, for regularizing large fully-connected layers within neural networks. When training with Dropout, a randomly selected subset of activations are set to zero within each layer. DropConnect instead sets a randomly selected subset of weights within the network to zero. Each unit thus receives input from a random subset of units in the previous layer. We derive a bound on the generalization performance of both Dropout and DropConnect. We then evaluate DropConnect on a range of datasets, comparing to Dropout, and show state-of-the-art results on several image recoginition benchmarks can be obtained by aggregating multiple DropConnect-trained models.

## 논문(Paper)에서 다루는 핵심 개념 (Idea) 💡
* DropConnect, dropout

## 논문(Paper) 정리 
* https://velog.io/@mingqook/Regularization-of-Neural-Networks-using-DropConnect

## 논문(Paper) 원본
* http://proceedings.mlr.press/v28/wan13.html?ref=https://githubhelp.com
